index.html

```diff
@@ -0,0 +1,206 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500;600;700;800&family=Roboto:wght@300;400;500&display=swap"
+    rel="stylesheet" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"
+    integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <div class="after-bdy">
```

Comment:
I have observed that the height of your page is smaller than the Figma design.
That's why the space highlighted here is smaller in your design too.
![image](https://user-images.githubusercontent.com/61361037/219710472-6440f08e-24bf-49df-8cbd-abb1706fa684.png)

Here is the comparison of your desing.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219709828-b33dfc62-fe01-421d-83a6-f5dba7891500.png)  | ![image](https://user-images.githubusercontent.com/61361037/219709953-ffbc572e-fcb6-4abb-9340-66a1f5fc60b3.png)  |